iscservice/ut/service_ut.c

```diff
@@ -563,7 +569,11 @@ static uint32_t remote_invocation_async(int exp_rc, uint32_t buf_type,
 		arg_fop[0] = m0_alloc(sizeof arg_fop[0][0]);
 		M0_UT_ASSERT(arg_fop[0] != NULL);
 		m0_fop_init(arg_fop[0], &m0_fop_isc_fopt,
-			    &remote_call_info.riv_req, m0_fop_release);
+			    NULL, m0_fop_release);
+		rc = m0_fop_data_alloc(*arg_fop);
```

Comment:
I think we can use `arg_fop[0]` here, instead of `*arg_fop`,
because it is used elsewhere.

Reply:
Addressed in new PR raised against main branch.